source/extensions/filters/http/cache/cacheability_utils.h

```diff
@@ -15,13 +15,15 @@ class CacheabilityUtils {
   // This does not depend on cache-control headers as
   // request cache-control headers only decide whether
   // validation is required and whether the response can be cached.
-  static bool isCacheableRequest(const Http::RequestHeaderMap& headers);
+  static bool CanServeRequestFromCache(const Http::RequestHeaderMap& headers);
```

Comment:
For functions we use camelCased names. Envoy's coding style diverges from Google's here.